multiversx_sdk/abi/abi.py

```diff
@@ -250,6 +250,24 @@ def decode_event(self, event_name: str, topics: list[bytes], additional_data: li
 
         return result
 
+    def encode_custom_type(self, name: str, values: list[Any]):
```

Comment:
I wonder if using `_get_custom_type_prototype()` would have been better :thought_balloon:

Reply:
Not sure. We can think about it. I'll merge it as it is and we'll see.